step-maven-plugin/step-commons-maven-plugin/src/main/java/step/plugins/maven/AbstractRunDeployedExecutionBundleMojo.java

```diff
@@ -0,0 +1,52 @@
+package step.plugins.maven;
+
+import org.apache.maven.plugin.MojoExecutionException;
+import org.apache.maven.plugins.annotations.Parameter;
+import step.core.repositories.RepositoryObjectReference;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public abstract class AbstractRunDeployedExecutionBundleMojo extends AbstractRunExecutionBundleMojo {
```

Comment:
ExecutionBundles have been renamed AutomationPackages. Please rename all the classes accordingly